uploader.json

```diff
@@ -1,5 +1,10 @@
 {
     "commands": [
+        {
+            "command": "/usr/sbin/dmidecode -s system-uuid",
+            "pattern": [],
+            "symbolic_name": "bios_uuid"
+        },
```

Comment:
If these items won't be collected by the v1 client then you don't need to add them to `uploader.json`.